index.html

```diff
@@ -159,6 +159,48 @@ <h5>LET’S CONNECT</h5>
           </li>
         </ul>
       </section>
+      <section class="flex leg">
+        <footer>
+          <h2 id="contact-me">Contact me</h2>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form
+            class="contact flex"
+            action="https://formspree.io/f/xrgjaknz"
+            method="post"
+          >
+            <input
+              type="text"
+              placeholder="Name"
+              id="name"
+              required
+              name="user_name"
+              maxlength="30"
+            />
+            <input
+              type="email"
+              placeholder="email"
+              id="email"
+              required
+              name="user_email"
+            />
+            <textarea
+              name="user_message"
+              id="msg"
+              cols="30"
+              rows="10"
+              placeholder="Write your message here"
+              required
+              maxlength="500"
+            ></textarea>
+            <button id="button" type="submit">Get in touch</button>
+            <div></div>
+          </form>
+        </footer>
+      </section>
```

Comment:
- [ ] Kindly make sure to have this section display properly on width "375px". This section has excess width due to using fixed widths for containers which is not recommended.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/160912795-36825138-1002-406f-b662-b859661caaa3.png) | ![image](https://user-images.githubusercontent.com/62290545/160912865-34fe21ac-73e2-4dfc-b82d-f738725d6979.png) |

Reply:
- [ ] The problem still exists so Please fix it.
[Hint]: you can fix it by giving each container of every section inside your site percentage width value and then you can center it for example:
`width: 80%;`
`margin: 0 auto;`